vars/buildExecute.groovy

```diff
@@ -102,7 +108,12 @@ void call(Map parameters = [:]) {
                     error "[${STEP_NAME}] buildTool not set and no dockerImage & dockerCommand provided."
                 }
         }
-
+        if (config.buildTool != 'docker' && config.cnbBuild) {
+            if (!['npm', 'maven', 'mta', 'docker'].contains(config.buildTool)) {
```

Comment:
`gradle` as `buildTool` is missing .

Reply:
Does it need to be there? It's not handled above- nothing happens with a gradle build in buildExecute. But I can add it if it's needed